src/lib/falcor/routes/articles/__tests__/database-calls.test.js

```diff
@@ -23,12 +23,12 @@ const database = getDatabaseConnection(databaseName);
 const maxSlugNum = 200;
 
 // End the knex connection after all tests are done
-afterAll(() => database.destroy());
+afterAll(async () => database.destroy());
```

Comment:
This is just because I found out that returning a promise from an async function automatically unwraps the promise and returns the value it resolves, and then I think this is more explicit. It has no actual semantic difference though